deploy/fabfile.py

```diff
@@ -33,26 +33,38 @@
 home = "/home/site"
 log = "%s/hookshot.log" % home
 current = "%s/%s/current" % (home, environment)
+ruby = "/opt/install/rbenv/shims/ruby"
 
-# principal command to run upon update
-command = "cd %s && git pull && bundle exec jekyll build >> %s" % (current, log)
+if environment == 'staging':
+  # on staging, build the site but also push back the extracted data files
+  command = "cd %s && git pull && %s _data/import-data && git add _data/team.yml _data/projects.yml && git commit -m 'update team and project data' && git push origin staging && %s go build >> %s" % (current, ruby, ruby, log)
```

Comment:
Also, @gboone, remember how I once added this to the `./go` deployment command:

```shell
if git commit -m 'update team and project data'; then git push origin staging; fi
```

Otherwise `git commit` will fail and `go build` won't run.